src/main.ts

```diff
@@ -327,6 +329,61 @@ export async function getExecutionGraphResult(
   }
 }
 
+export async function prettifyExecutionGraphResult(
+  executionGraphResult: Object
+): Promise<void> {
+  core.debug(`Execution Graph Result: ${JSON.stringify(executionGraphResult)}`)
+  core.info(
+    c.bold(
+      `Execution Graph Result: ${
+        executionGraphResult["passed"] ? "passed" : "failed"
+      }`
+    )
+  )
+  let actionsPassed = 0
+  let actionsFailed = 0
+  let actionsSkipped = 0
+  for (const task of executionGraphResult["actions"]) {
```

Comment:
The lint is giving you a hint here that we are not handling skipped actions. You should check with the team how to check if an action is skipped so you can count it.